File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.py
import sys

standard = sys.stdin.readline().split()
n = int(standard[0])
x = int(standard[1])

input = sys.stdin.readline().split()
output = []
for i in range(n):
    if x > int(input[i]):
        output.append(input[i])

print(' '.join(output))

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const fs = require("fs");
const inputData = fs.readFileSync("/dev/stdin").toString().trim().split("\n");
const [N, X] = inputData[0].split(" ").map(Number);
const nums = inputData[1].split(" ").map(Number);

const answers = nums.filter((num) => num < X);
console.log(answers.join(" "));
